.NET/Microsoft.Recognizers.Text.DataTypes.DataDrivenTests/TestTimexResolver.cs

```diff
@@ -525,40 +525,33 @@ public void DataTypes_Resolver_DateTime_TuesAt12PM_UtcInput()
             Assert.AreEqual(nextWeekUtc.ToString("yyyy-MM-dd HH:mm:ss"), resolution.Values[1].Value);
             Assert.IsNull(resolution.Values[1].Start);
             Assert.IsNull(resolution.Values[1].End);
-        }
-
+        }
+
         [TestMethod]
-        public void DataTypes_Resolver_DateTime_Weekend()
```

Comment:
We should keep the weekend and un-anchored month unit tests as well. Can you restore them?
We always suggest appending new tests at the end.